bokehjs/src/lib/core/visuals.ts

```diff
@@ -128,8 +123,6 @@ function create_hatch_canvas(hatch_pattern: mixins.HatchPattern, hatch_color: Co
       ctx.stroke()
       ctx.stroke()
```

Comment:
@bryevdv, why double stroke this path?

Reply:
If I didn't leave a comment it was probably just a mistake. That said, I'd make an issue to investigate rather than remove here, just in case